site/content/docs/5.2/examples/features/index.html

```diff
@@ -285,4 +285,63 @@ <h4 class="fw-bold mb-0">Featured title</h4>
       </div>
     </div>
   </div>
+
+  <div class="b-example-divider"></div>
+
+  <div class="container px-4 py-5">
+    <h2 class="pb-2 border-bottom">Features with Title</h2>
+
+    <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-4 py-5">
```

Comment:
Changing to `g-5` for more space.

```suggestion
    <div class="row row-cols-1 row-cols-md-2 align-items-md-center g-5 py-5">
```